projects/elife.yaml

```diff
@@ -2050,6 +2050,17 @@ kubernetes-aws:
     description: project managing an EKS cluster
     domain: False
     intdomain: False
+    terraform:
+        version: "0.13.7" # next version is "0.14.11" see `install-terraform.sh`
+        provider-aws:
+            source: "hashicorp/aws"
+            version: "4.66.1"
+        provider-tls:
+            source: "hashicorp/tls"
+            version: "4.0.4"
+        provider-kubernetes:
+            source: "hashicorp/kubernetes"
+            version: "2.20.0"
```

Comment:
These three providers are only used by `kubernetes-aws` so their global configuration can be changed in the `defaults` section.

I'll do this today.

Reply:
Done.